internal/blocks/blocks.go

```diff
@@ -0,0 +1,90 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package blocks
+
+import (
+	"bytes"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/onflow/flow-cli/internal/events"
+
+	"github.com/onflow/flow-go-sdk"
+	"github.com/onflow/flow-go-sdk/client"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:              "blocks",
+	Short:            "Utilities to read blocks",
+	TraverseChildren: true,
+}
+
+// BlockResult
+type BlockResult struct {
+	block       *flow.Block
+	events      []client.BlockEvents
+	collections []*flow.Collection
+	verbose     bool
+}
+
+// JSON convert result to JSON
+func (r *BlockResult) JSON() interface{} {
+	return r
```

Comment:
All the struct fields are private, so I think they'll be ignored by the JSON marshaler